docs/modding/tools.md

```diff
@@ -0,0 +1,70 @@
+# Modding Tools
+
+## Source Engine
+
+### Titanfall
+
+* [Titanfall VPK Tool](http://cra0kalo.com/public/Titanfall\_VPKTool3.4\_Portable.zip)
+* [Legion](https://wiki.modme.co/wiki/apps/Legion.html)
+
+### RPSNVPK's
+
+* [squidgyberries RSPNVPK](https://github.com/squidgyberries/RSPNVPK)
+* [taskinoz RSPNVPK](https://github.com/taskinoz/RSPNVPK)
+* [Provoxin RSPNVPK](https://github.com/Provoxin/RSPNVPK-GUI)
+
+{% embed url="https://noskill.gitbook.io/titanfall2/how-to-start-modding/modding-introduction/modding-tools/how-to-use-rspnvpk" %}
+NoSkill: How to use RSPNVPKs
+{% endembed %}
+
+### VTF & VMT
+
+* [VTFEdit](this-content-is-no-longer-available.md)
```

Comment:
I guess the link broke while editing? ^^

Reply:
That is why it's present in .github/tools/ but might see if there's an archive.